yarn-project/end-to-end/src/devnet/e2e_smoke.test.ts

```diff
@@ -123,18 +123,19 @@ describe('End-to-end tests for devnet', () => {
     } else if (PXE_URL) {
       logger.info(`Using PXE_URL: ${PXE_URL}`);
       pxe = createPXEClient(PXE_URL);
+      node = createAztecNodeClient(PXE_URL);
```

Comment:
this is weird but both services share the URL here. This will get resolved with me dropping the PXE JSON RPC